exporter/sematextexporter/README.md

```diff
@@ -0,0 +1,58 @@
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@AkhigbeEromo](https://www.github.com/AkhigbeEromo), [@boratanrikulu](https://www.github.com/boratanrikulu), [@akshatagarwl](https://www.github.com/akshatagarwl) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+This exporter supports sending metrics to [Sematext Cloud](https://sematext.com/) in Influx line protocol format and logs using the Bulk Index Api format.
+
+## Configuration
+
+The following configuration options are supported:
+* `timeout` (default = 5s) Timeout for requests
+* `region`  Region specifies the Sematext region the user is operating in; must be one of:
+  * `US`
+  * `EU`
+* `app_token` specifies the token of the Sematext Monitoring/Logs App to which the user wants to send data. It must be a valid UUID string in the format `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`. For example: `2064e37c-4fac-45f6-831d-922d43fde759`.
+* `payload_max_lines` (default = 1_000) Maximum number of lines allowed per HTTP POST request
+* `payload_max_bytes` (default = 300_000) Maximum number of bytes allowed per HTTP POST request
+* `metrics_schema` (default = telegraf-prometheus-v2) The chosen metrics schema to write
+* `sending_queue` [details here](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.25.0/exporter/exporterhelper/README.md#configuration)
+    * `enabled` (default = true)
+    * `num_consumers` (default = 10) The number of consumers from the queue
+    * `queue_size` (default = 1000) Maximum number of batches allowed in queue at a given time
+* `retry_on_failure` [details here](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.25.0/exporter/exporterhelper/README.md#configuration)
+    * `enabled` (default = true)
+    * `initial_interval` (default = 5s) Time to wait after the first failure before retrying
+    * `max_interval` (default = 30s) Upper bound on backoff interval
+    * `max_elapsed_time` (default = 120s) Maximum amount of time (including retries) spent trying to send a request/batch
+
+The full list of settings exposed for this exporter are documented in [config.go](config.go).
+
+Example:
+```yaml
+timeout: 500ms
+region: US  
+retry_on_failure:
+  enabled: true
+  initial_interval: 1s
+  max_interval: 3s
+  max_elapsed_time: 10s
+metrics:
+  app_token: 2064e37c-4fac-45f6-831d-922d43fde759
```

Comment:
This example does not align with the Configuration section above. The configuration section describes a top-level property `app_token`, but the example has `metrics::app_token` and `logs_app_token`.

Reply:
The `payload_*` properties are still wrong.

Perhaps we can remove all this config for now as described in [another comment](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37161/files#r1913196218)?